src/index.html

```diff
@@ -8,6 +8,84 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Stars</h1>
+    <div class="container">
+      <div class="stars stars--0">
+        <span class="stars__star">
+        </span>
```

Comment:
No need to do it, it's short enough. And it's better to make them `div`s, `span` is used mostly for text.
```suggestion
        <span class="stars__star"></span>
```